default.nix

```diff
@@ -42,6 +42,8 @@ let
         hnix-store-core = self.callHackage "hnix-store-core" "0.1.0.0" {};
 
         ghcid = self.callCabal2nix "ghcid" (hackGet ./dep/ghcid) {};
+        # Exports more internals
```

Comment:
Should we put the PR link here?

Reply:
We're tracking this as a separate ticket.